src/components/TabTraining.vue

```diff
@@ -1,6 +1,13 @@
 <template>
   <div class="tab training">
     <h1>Training Options</h1>
```

Comment:
Let's add a bit of text between `<h1>Training Options</h1>` and `<h2>Argument Parser</h2>` explaining what argument parser is about and what to choose.
```
<h1>Training Options</h1>

We are using yaml files to set up the training configuration (see config.yaml). 
Additional arguments given to the main script can be parsed using one of the tools provided below:

<h2>Argument Parser</h2>

...

* [argparse](link to python) - is python built-in tool to handler cmd arguments
* [python-fire](link to fire docs) - is a tool bla-bla-bla why it is nice in one sentence
* [hydra](link to hydra) - bla-bla-bla why it is nice in one sentence

<h2 class="training">Deterministic Training</h2>
```